acceptance_tests/features/printfile.feature

```diff
@@ -1,8 +1,8 @@
 Feature: Print files can be created and sent with correct data
 
-  Scenario Outline: A case is loaded, wave of contact triggered and print file created with differing templates with UACs
+  Scenario Outline: A case is loaded, action rule triggered and print file created with differing templates with UACs
     Given sample file "<sample file>" is loaded successfully
-    When a print wave of contact has been created with template "<template>" and classifiers "1=1"
+    When a print action rule has been created with template "<template>" and classifiers "1=1"
```

Comment:
For these print rules I would like to include the "default" classifiers which are no longer hard coded in the case processor, to make the example.

Reply:
classifiers are an action rule feature, not a print file feature. I propose having a different ticket to show off the power of classifiers